src/components/MultiSelect/FilterableMultiSelect.js

```diff
@@ -216,7 +257,12 @@ export default class FilterableMultiSelect extends React.Component {
                 <ListBox.Field {...getButtonProps({ disabled })}>
                   {selectedItem.length > 0 && (
                     <ListBox.Selection
-                      clearSelection={clearSelection}
+                      clearSelection={e => {
+                        {
+                          clearSelection(e);
```

Comment:
Can `clearSelection` be refactored to include the `handleClearSubOptions`? Is there ever a scenario where you'd want the selection cleared but the subOptions not? If there isn't I'd think it should be done in the same function.